tarball/tarfile.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package tarball provides functions to manipulate tar files.
```

Comment:
Mark it as implementation detail - if somebody complains, we can probably export it as a `pkg` subpackage?